app/views/main_content_panel.py

```diff
@@ -17,6 +17,9 @@
 from typing import Any, Callable, Self
 from urllib.parse import urlparse
 from zipfile import ZipFile
+from pathlib import Path
+from app.utils.generic import platform_specific_open
+from app.utils.system_info import SystemInfo
 
 from loguru import logger
```

Comment:
Needs to be auto-sorted with Ruff

Reply:
Oo, I thought `ruff format` would do this. Fixed using `ruff check --fix`